ios/FluentUI/Vnext/Notification/NotificationView_SwiftUI.swift

```diff
@@ -45,41 +42,98 @@ import SwiftUI
 /// View that represents the Notification.
 public struct NotificationViewSwiftUI: View, ConfigurableTokenizedControl {
     @Environment(\.horizontalSizeClass) var horizontalSizeClass: UserInterfaceSizeClass?
-    @Environment(\.swiftUIInsets) private var safeAreaInsets: EdgeInsets
     @Environment(\.fluentTheme) var fluentTheme: FluentTheme
     @ObservedObject var state: MSFNotificationStateImpl
+    @Binding var isPresented: Bool
+    @State private var dismissedOffset: CGFloat = 0
+    @State private var yOffsetFromBottom: CGFloat = 0
```

Comment:
nit: I'm thinking that **bottomOffset** is a simpler name for this state var (since bottom implies that it's the Y coordinate).

On the same logic, I think the variable below could be something like **bottomOffsetForDismissedState** or something like that?

Thoughts?

Reply:
Sounds good to me!